connect/runtime/src/test/java/org/apache/kafka/connect/runtime/isolation/TestPlugins.java

```diff
@@ -64,56 +63,95 @@
  * the names of the different plugins directly via the exposed constants.
  */
 public class TestPlugins {
+    public enum TestPlugin {
+        /**
+         * Resource dir and class name of a plugin which will always throw an exception during loading
```

Comment:
The "Resource dir and class name of a..." prefix for each of these is a little redundant. Can we nix that and just use, e.g., `A plugin which will always throw an exception during loading`, `A {@link org.apache.kafka.common.config.provider.ConfigProvider} which samples information about its method calls.`, etc.?